includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,219 @@
+
+		return [
+			'tasks_list'  => [
+				'tasks' => [
+					// TODO: Generate based on Setup Wizard data + site info.
+					[
+						'title' => 'Set up Course Site',
+						'done'  => true,
+						'url'   => null,
+					],
+					[
+						'title' => 'Create your first Course',
+						'done'  => false,
+						'url'   => '/wp-admin/edit.php?post_type=course',
+					],
+					[
+						'title' => 'Configure Learning Mode',
+						'done'  => false,
+						'url'   => '/wp-admin/edit.php?post_type=course&page=sensei-settings#course-settings',
+					],
+					[
+						'title' => 'Publish your first Course',
+						'done'  => false,
+						'url'   => '???',
+					],
+				],
+				'image' => 'http://...', // For the moment we are returning the task_list image url from the backend. We could see how to calculate this on frontend based on tasks response.
+			],
+			'quick_links' => [
+				// TODO: Generate or hardcode in frontend.
+				'Courses'           => [
+					[
+						'title' => 'Getting Started Guide',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Create a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Import a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Reports',
+						'url'   => 'http://...',
+					],
+				],
+				'Settings'          => [
+					[
+						'title' => 'Email notifications',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Learning Mode',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'WooCommerce',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Content Drip',
+						'url'   => 'http://...',
+					],
+				],
+				'Advanced Features' => [
+					[
+						'title' => 'Interactive Blocks',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Groups & Cohorts',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Quizzes',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Integrations',
+						'url'   => 'http://...',
+					],
+				],
+			],
+			'help'        => [
+				// TODO: Generate or hardcode in frontend.
+			],
+			'guides'      => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'        => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31T21:41:38',
```

Comment:
Just a wee note that this is the format (RFC3339) used on the SenseiLMS.com endpoint but we'll want to localize it (probably just the date and ignore the time) before sending to frontend.

Reply:
Clarified in d5d096198